documentation/book/ref-kafka-connect-configuration.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect-configuration.adoc
+
+[id='ref-kafka-connect-configuration-{context}']
+= Kafka Connect configuration
+
+Kafka Connect can be configured using the `config` property in `KafkaConnect.spec` and `KafkaConnectS2I.spec`.
+This property should contain the Kafka Connect configuration options as keys.
+The values could be in one of the following JSON types:
+
+* String
+* Number
+* Boolean
+
+Users can specify and configure the options listed in {ApachaKafkaConnectConfig} with the exception of those options which are managed directly by {ProductName}.
```

Comment:
listed in {ApachaKafkaConnectConfig} with > listed in the {ApachaKafkaConnectConfig} with